go.mod

```diff
@@ -0,0 +1,41 @@
+module github.com/open-policy-agent/kube-mgmt
+
+go 1.17
```

Comment:
Let's use `1.17.6`

Reply:
> Thanks for submitting this PR. This is a great change. We need to make sure the build and release process do not break. The build seems to be ok. It would be great if you could release a new version of the project on your fork to test this out.

Thanks very much for reviewing this PR. I have modified the `.github/workflows/release.yaml` file in a [branch forked from this PR](https://github.com/warpcomdev/kube-mgmt/tree/release/0.0.1) to set the `env.image` variable to `warpcomdev/kube-mgmt` instead of `openpolicyagent/kube-mgmt`, and released dummy version 0.0.1.

- The release action completed successfully (https://github.com/warpcomdev/kube-mgmt/actions/runs/1765805717)
- The chart was compiled and published in the `gh-pages` branch (https://github.com/warpcomdev/kube-mgmt/tree/gh-pages)

Regarding go 1.17, changing the line in `go. mod` to *qo 1.17.6* breaks `go mod tidy`:

```bash
$ go mod tidy
go: errors parsing go.mod:
/home/rafa/projects/kube-mgmt/go.mod:3: invalid go version '1.17.6': must match format 1.23
```

What I did instead was to change the *FROM* line in the `Dockerfile` to read *FROM golang:1.17.6-alpine as build*, in commit 3e172c0

Reply:
Regarding go 1.17, changing the line in `go.mod` to *qo 1.17.6* breaks `go mod tidy`:

```bash
$ go mod tidy
go: errors parsing go.mod:
/home/rafa/projects/kube-mgmt/go.mod:3: invalid go version '1.17.6': must match format 1.23
```

What I did instead was to change the *FROM* line in the `Dockerfile` to read *FROM golang:1.17.6-alpine as build*, in commit 3e172c0